src/main/kotlin/org/team5419/frc2019offseason/subsystems/Vision.kt

```diff
@@ -103,6 +103,10 @@ public class Vision : Subsystem() {
         timer.start()
     }
 
+    public fun flashOnce() {
+        flashForSeconds(Constants.Vision.BLINKS_PER_SECOND)
+    }
+
```

Comment:
I think that when we pick up the game piece we want to flash a few times, like maybe three times